### ISSUE TITLE
%use behavior

### ISSUE THREAD

**Issue by jma7** [vatlab/SoS#1193]
I was just following the tutorials, when I try
```
   %use R
```
It seems only the current cell shift to R, the new cell is still in SoS. Is this supposed to be?

**Comment by BoPeng** [comment]
Yes, there is no longer a concept of "global default", new cell just use the kernel before it.

The `%use` magic is simply a new advanced version (allow parameters) of starting new kernels.
